### PR TITLE
Murphant community 1

### PR DESCRIPTION
**Pull request by LeMurphant**
As with my other PR, the lint fails.

Here are the TODOs that are left
        {/* TODO make this only appear in left part of page */}
        {/* TODO make this only appear in left part of page */}
        {/* TODO there seems to be lacking text or a link in the figma */}
          // TODO: do we want to link to a specific tag here?

### CODE REVIEW

**Review by bryceerobertson — APPROVED** [review]

File: app/routes/how-can-i-help.community.tsx [hunk 1]
```diff
@@ -1,14 +1,141 @@
...
 }
 
+const InPerson = () => (
+  <>
+    <div className="padding-bottom-24">
+      <div className="flexbox padding-bottom-56">
+        <h2 className="teal-500 padding-bottom-40" id="new-to-aisafety">
+          In person communities
+        </h2>
+        <p className="grey default">
```
Comment (melissasamworth) [inline app/routes/how-can-i-help.community.tsx:21]: To make it just half the page, you would apply the  col-6 class to this p

File: app/routes/how-can-i-help.community.tsx [hunk 2]
```diff
@@ -1,14 +1,141 @@
...
+        title="AI safety-specific coommunities"
+        links={[
+          {
+            title: 'Our list of in-person communities',
+            action: 'https://www.aisafety.com/communities',
+          },
+        ]}
+      >
+        <p className="grey default padding-bottom-24">
+          There are about 60 in-person AI Safety communities across the globe, mostly in major
```
Comment (melissasamworth) [inline app/routes/how-can-i-help.community.tsx:37]: safety*

File: app/routes/how-can-i-help.community.tsx [hunk 3]
```diff
@@ -1,14 +1,141 @@
...
+)
+
+const Online = () => (
+  <>
+    <div className="padding-bottom-24">
+      <div className="flexbox padding-bottom-56">
+        <h2 className="teal-500 padding-bottom-40" id="new-to-aisafety">
+          Online communities
+        </h2>
+        <p className="grey default">
```
Comment (melissasamworth) [inline app/routes/how-can-i-help.community.tsx:71]: add class col-6

File: app/routes/how-can-i-help.community.tsx [hunk 4]
```diff
@@ -1,14 +1,141 @@
...
+        <p className="grey default padding-bottom-24">
+          General hubs for support, motivation, discussion, and information-gathering.
+        </p>
+      </HelpItem>
+
+      <HelpItem title="Focused on one activity">
+        <p className="grey default padding-bottom-24">
+          Such as volunteering for a certain project or organizing and attending protests.
+        </p>
+        {/* TODO there seems to be lacking text or a link in the figma */}
```
Comment (melissasamworth) [inline app/routes/how-can-i-help.community.tsx:105]: I hadn't included one but I like your suggestion, why not include something. Can you think of the best community to put here that fits this description?

File: app/routes/how-can-i-help.community.tsx [hunk 5]
```diff
@@ -1,14 +1,141 @@
...
+
+      <HelpItem
+        title="Detailed strategic discussion"
+        links={[
+          {
+            tag: 'Top recommendation',
+            title: 'Lesswrong',
+            action: 'https://www.lesswrong.com/',
+          },
+          // TODO: do we want to link to a specific tag here?
```
Comment (melissasamworth) [inline app/routes/how-can-i-help.community.tsx:116]: A specific lesswrong tag? Sure, if you think that would make sense. You can choose what tag we link to

**Review by melissasamworth — CHANGES_REQUESTED** [review]

File: app/routes/how-can-i-help.community.tsx [hunk 6]
```diff
@@ -1,14 +1,141 @@
...
 export const meta: MetaFunction = () => {
   return [{title: 'How Can I Help? - AISafety.info'}]
 }
 
+const InPerson = () => (
+  <>
+    <div className="padding-bottom-24">
+      <div className="flexbox padding-bottom-56">
+        <h2 className="teal-500 padding-bottom-40" id="new-to-aisafety">
+          In person communities
```
Comment (bryceerobertson) [inline app/routes/how-can-i-help.community.tsx:19]: Should be in-person

File: app/routes/how-can-i-help.community.tsx [hunk 7]
```diff
@@ -1,14 +1,141 @@
...
+        </h2>
+        <p className="grey default">
+          In person communities provide general purpose support, networking, and socializing for
```
Comment (bryceerobertson) [inline app/routes/how-can-i-help.community.tsx:22]: In-person

File: app/routes/how-can-i-help.community.tsx [hunk 8]
```diff
@@ -1,14 +1,141 @@
...
+            action: 'https://forum.effectivealtruism.org/groups',
+          },
+          {
+            title: 'Rationalist groups map',
+            action: 'https://www.lesswrong.com/community',
+          },
+        ]}
+      >
+        <p className="grey default padding-bottom-24">
+          AI Safety is a major topic in Effective Altruism (EA) and Rationalism. Consider joining a
```
Comment (bryceerobertson) [inline app/routes/how-can-i-help.community.tsx:56]: "AI safety"

File: app/routes/how-can-i-help.community.tsx [hunk 9]
```diff
@@ -1,14 +1,141 @@
...
+          local EA or Rationalist group if you can't find an AI Safety community near you.
```
Comment (bryceerobertson) [inline app/routes/how-can-i-help.community.tsx:57]: "AI safety"

**Review by bryceerobertson — APPROVED** [review]

File: app/routes/how-can-i-help.community.tsx (hunk 1 above)
Comment (LeMurphant) [reply app/routes/how-can-i-help.community.tsx:21]: Yep, that worked

File: app/routes/how-can-i-help.community.tsx (hunk 2 above)
Comment (LeMurphant) [reply app/routes/how-can-i-help.community.tsx:37]: I was not sure about this one, but Claude is on your side so I will change it. See how quickly our mistrust of AI erodes! Claude knows best!

File: app/routes/how-can-i-help.community.tsx (hunk 4 above)
Comment (LeMurphant) [reply app/routes/how-can-i-help.community.tsx:105]: I added a (hedged) blurb on PauseAI, lmk what you think

File: app/routes/how-can-i-help.community.tsx (hunk 5 above)
Comment (LeMurphant) [reply app/routes/how-can-i-help.community.tsx:116]: I added the "AI" tag

File: app/routes/how-can-i-help.community.tsx (hunk 2 above)
Comment (melissasamworth) [reply app/routes/how-can-i-help.community.tsx]: 🙃

File: app/routes/how-can-i-help.community.tsx (hunk 4 above)
Comment (melissasamworth) [reply app/routes/how-can-i-help.community.tsx]: Pause AI discord as a focused activity community? I like that. Might change the copy later.

**Review by melissasamworth — APPROVED** [review]

File: app/routes/how-can-i-help.community.tsx (hunk 4 above)
Comment (LeMurphant) [reply app/routes/how-can-i-help.community.tsx]: I linked directly to their website on the events page since this part suggests in-person stuff.

**Pull request merged**